file: fix install section

ratkaj opened the pull request:
Maintainer: Marko Ratkaj <marko.ratkaj@sartura.hr> / @ratkaj
Compile tested: LEDE master, mvebu
Run tested: LEDE master, mvebu, Linksys wrt3200

Mantainer changed to me with permission from Luka Perkov @lperkov 

Description:
By using $(CP) instead of $(INSTALL_DATA) we preserve symbolic links and
by doing so we don't copy the same file multiple times. This saves some
space.

champtar commented:
please increase PKG_RELEASE

ratkaj commented:
@champtar thank you for reminding me of that. I might as well update to the latest v5.32 instead. I will update the pull request tomorrow.

diizzyy commented:
Can you also please add a https and http mirror such as the following as ftp might be troublesome in some corp environments (we're already doing this for aliases in download.pl (LEDE) script)?

https://mirror.csclub.uwaterloo.ca/gentoo-distfiles/distfiles/
http://mirrors.rit.edu/gentoo/distfiles/
<main site>

ratkaj commented:
Since version 5.32 does not work and needs more debugging i updated PKG_RELEASE and i added https and http sources. @diizzyy v5.25 is not in repos you recommended so i had to find others.

Will debugg 5.32 when i find the time

Thank you all for your feedback
